src/emissions-workspace/emissions.service.ts

```diff
@@ -76,6 +77,7 @@ export class EmissionsWorkspaceService {
     const DAILY_EMISSION = 2;
     const SORBENT_TRAP = 3;
     const WEEKLY_TEST_SUMMARIES = 4;
+    const SUMMARY_VALUES = 5;
```

Comment:
This value needs to be unique and correlate to its position in the promise array.

Issue probably caused by merge.